lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -239,7 +239,7 @@ spec = describe "SHELLEY_MIGRATIONS" $ do
             Empty >>= flip verify
             [ expectField
                 #distribution
-                ((`shouldBe` (Just 100)) . Map.lookup 100_000_000_000)
+                ((`shouldBe` (Just 99)) . Map.lookup 100_000_000_000)
```

Comment:
So there's one less... I'm suspecting because it got merged with the dust coins, which are now properly treated as dust?

Reply:
Just to make sure, could you control what the distribution looks like? (you know, if we've lost 100K Ada along the way, it is quite a problem ^.^)

Reply:
This is the distribution:

```
foo = Just
  (fromList
    [ (10               , 0)
    , (100              , 0)
    , (1000             , 0)
    , (10000            , 0)
    , (100000           , 0)
    , (1000000          , 0)
    , (10000000         , 1)
    , (100000000        , 6)
    , (1000000000       , 0)
    , (10000000000      , 0)
    , (100000000000     , 99)
    , (1000000000000    , 1)
    , (10000000000000   , 0)
    , (100000000000000  , 0)
    , (1000000000000000 , 0)
    , (10000000000000000, 0)
    , (45000000000000000, 0)
    ]
  )
```